src/_includes/common/configuration-shutdown-draining.md

```diff
@@ -0,0 +1,22 @@
+<!--
+Guideline: This page is common to all SDKs; it is stored in the common folder, nested under _includes/common. To use, 
+
+1. Add a folder with the name of the platform you are documenting to the _documentation/sdks structure (for example, _documentation/sdks/javascript) 
+2. Create a new shutdown-drain.md file in _documentation/sdks/<platform-name>/config directory 
+3. Create the defined `include` statements and add them to the your new file
+
+If you have questions, please ask Fiona or Daniel. 
+-->
```

Comment:
These comments will show up in the source code, maybe we'll find a better place for them or use `{% comment %}` `{% endcomment %}` instead of `<!--` `-->`

Reply:
Fixed. Thanks.